[PROCEDURES] Officially allow adding new datatypes to the latest tagged release (#9625)
galaxyproject/galaxy: doc/source/project/organization.rst

membership to the *committers* group - as such new members may be added to
this group by opening a pull request adding a qualified person to this list.
Pull requests modifying the membership of this list are subject to the normal
rules for pull requests that modify governance procedures outlined below, with
one exception: a *committer* may not vote
against their own removal from the group (for obvious reasons).

Given the responsibilities and power invested in this group, it is important

> that individuals not actively working on Galaxy any more are removed from

Reviewer: ```suggestion
that individuals not actively working on Galaxy anymore are removed from
```

the group. If individuals in this group intend to change jobs or reallocate
volunteer activities and will no longer be active in the Galaxy community,
they should withdraw from membership of this group. Periodically, active

> members may review this group and request inactive members are removed - this

Reviewer: ```suggestion
members may review this group and request that inactive members are removed - this
```
